### DISCUSSION TITLE
Multiple instances?

### DISCUSSION THREAD

**Discussion by pcmid** [immich-app/immich#4107] [Q&A]
Thanks to Alex and all other developers make so great project!

Recently I am trying to deploy immich on k8s, cound I set the replicas more than 1? 
I found out that the machine learning server will download modules to `/cache` dir, Is is safe for use common cache volume for multiple intances?

**Comment by bo0tzz** [comment]
Yup, that should be fine. If you run into issues from running multiple replicas please let us know.

**Reply by growse** [reply]
I'm testing a HorizontalPodAutoScaler for both the ml and microservices components. Seems to work ok so far...!
